docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -388,6 +388,62 @@ The default is `null`.
 Enables or disables P2P discovery.
 The default is `true`.
 
+### `ethstats`
+
+=== "Syntax"
+
+    ```bash
+    --ethstats=<nodename:secret@host:port>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    --ethstats=Dev-Node-1:secret@127.0.0.1:3001
+    ```
+
+=== "Environment Variable"
+
+    ```bash
+    BESU_ETHSTATS=Dev-Node-1:secret@127.0.0.1:3001
+    ```
+
+=== "Configuration File"
+
+    ```bash
+    ethstats="Dev-Node-1:secret@127.0.0.1:3001"
+    ```
+
+Reporting URL of an EthStats server.
+
+### `ethstats-contact`
```

Comment:
The contact seems to have an empty string default value and is optional. The `--ethstats` URL option is required if you want to use the contact option.